packages/chats/src/UIChatInput/ChatInput.tsx

```diff
@@ -48,11 +47,10 @@ export function ChatInput(props: ChatInputProps) {
         numberOfLinesProp,
         resetInputHeight,
     } = useAutogrowTextView(props.textInputRef, undefined, CHAT_INPUT_NUM_OF_LINES);
-    const showMaxLengthAlert = useChatMaxLengthAlert(MAX_INPUT_LENGTH);
     const { inputHasValue, onChangeText, onKeyPress, onSendText } = useChatInputValue({
         ref: props.textInputRef,
         onSendText: props.onSendText,
-        showMaxLengthAlert,
+        onMaxLength: props.onMaxLength,
```

Comment:
I'm curious why you decided to move it to the parent, instead of rendering notice here, could you explain your reasoning? 🤔 Seems like an extra step in configuring all that :)

Reply:
@savelichalex  Shouldn't we be able to control this Notice? If two or more notices appear in the same position at the same time then we will be able to control their state.

Reply:
Valid point :) What do you think to have by default it here, and only if a callback is provided give control to a user? 🤔 Just want to reduce things that are needed to be configured to use all that